doc/meetings/build-wg-meeting-2024-05-15.md

```diff
@@ -0,0 +1,56 @@
+# Node.js  Build WorkGroup Meeting 2024-05-15
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=CVmBq_wFd-8
+* **GitHub Issue**: https://github.com/nodejs/build/issues/3716
+* **Minutes Google Doc**: https://docs.google.com/document/d/1PFqf2H5gW12K92QEcoiDjH9Y2l_oFEZ-VOjEhtkZlN8/edit
+
+## Present
+
+* Ulises Gascon: @UlisesGascon
+* Michael Dawson: @mhdawson
+* Richard Lau: @richardlau
+* Moshe Atlow: @MoLow
+
+## Agenda
+
+## Announcements
+
+Github bot is currently not working. There is an issue with the dependencies. It will require some manual work as not everything can be done via Ansible. Note that this is a legacy service (Node@14). https://github.com/nodejs/build/issues/3720
```

Comment:
```suggestion
Github bot is currently not working. There is an issue with the dependencies. It will require some manual work as not everything can be done via the current Ansible scripts. Note that this is a legacy service (Node@14). https://github.com/nodejs/build/issues/3720
```